### PR TITLE
Optionally set Pro Rider equipment

### PR DESCRIPTION
**Pull request by daktak**
Using team name from procyclingstats.com, set riders bikes and wheels.

### CODE REVIEW

File: standalone.py [hunk 1]
```diff
@@ -578,6 +578,12 @@ def load_bots():
                             p.country_code = rider['country_code']
                             if 'jersey' in rider:
                                 p.ride_jersey = rider['jersey']
+                            if 'bike_frame' in rider:
+                                p.bike_frame = rider['bike_frame']
+                            if 'bike_wheel_front' in rider:
+                                p.bike_wheel_front = rider['bike_wheel_front']
+                            if 'bike_wheel_rear' in rider:
+                                p.bike_wheel_front = rider['bike_wheel_rear']
```
Comment (oldnapalm) [inline standalone.py:586]: Should be `p.bike_wheel_rear`

**Comment by oldnapalm** [comment]
I wrote this to create a teams dictionary using your script
[teams.zip](https://github.com/zoffline/zwift-offline/files/11834746/teams.zip)

This dictionary could be manually fixed and used by your script.

**Comment by oldnapalm** [comment]
I pushed my suggestion, the teams dictionary still has the fuzzy matches, needs manual fixing.

**Comment by daktak** [comment]
Yes, since zwift does not have jerseys for all the current teams (with sponsorship changes), fuzzy matching is required. I don't think its too bad to have a team wearing old jerseys.

I was using fuzzy for equipment as I was manually creating the bikes / wheels dictionary and didn't want to dig to get exact names from GameDictionary.

**Comment by daktak** [comment]
looks good, I like your simplification. And it makes sense to match proto.

**Pull request merged**